sdk/core/azure-core/tests/test_http_logging_policy.py

```diff
@@ -280,12 +296,14 @@ def emit(self, record):
     policy.on_response(request, response)
 
     assert all(m.levelname == 'INFO' for m in mock_handler.messages)
```

Comment:
Usage of `INFO` here seems inconsistent with the other code... 

Example: sdk/keyvault/azure-keyvault-certificates/tests/test_certificates_client.py